generators/docker-compose/files.js

```diff
@@ -40,7 +40,6 @@ function writeFiles() {
     writeKeycloakFiles() {
       if (this.authenticationType === OAUTH2 && this.applicationType !== MICROSERVICE) {
         this.template('realm-config/jhipster-realm.json.ejs', 'realm-config/jhipster-realm.json');
-        this.template('realm-config/jhipster-users-0.json.ejs', 'realm-config/jhipster-users-0.json');
```

Comment:
This file should be removed from existing projects.
They will fail if not removed.

Reply:
@mshima very good point I didn't think about it.

Though I am afraid removing it automatically would create problems for people who added users manually to that file and need to copy them to the other file.

What do you think ?

Reply:
We should remove it the user should have it inside a repository.